src/coreclr/scripts/superpmi_diffs.py

```diff
@@ -207,6 +207,8 @@ def main(main_args):
         os.path.join(script_dir, "superpmi.py"),
         "asmdiffs",
         "--no_progress",
+        "--metrics", "CodeSize",
+        "--metrics", "PerfScore",
```

Comment:
Are you doing this so we see PerfScore reports in the CI?

I am on the fence about this. I'd like to keep the default simple.

Reply:
Yes, I will revert this change before merging. This is just to see the PerfScore impact for all the collections/platforms.